utils/node_local_test.py

```diff
@@ -1132,7 +1132,7 @@ def import_daos(server, conf):
 def run_daos_cmd(conf,
                  cmd,
                  show_stdout=False,
-                 valgrind=True):
+                 valgrind=False):
```

Comment:
This is just now impossible I assume?  We could in theory simplify this function accordingly however it would make it harder in the future if we ever were able to run go under memcheck.